client/sites/components/site-preview-pane/preview-pane-header-buttons.tsx

```diff
@@ -35,8 +37,18 @@ const PreviewPaneHeaderButtons = ( { focusRef, itemData }: Props ) => {
 	const hasStagingSite = useSelector( ( state ) =>
 		hasWpcomStagingSite( state, itemData.blogId ?? null )
 	);
+	const stagingStatus = useSelector( ( state ) =>
+		getStagingSiteStatus( state, itemData.blogId ?? 0 )
+	);
+
+	const isStagingSiteProgressing =
```

Comment:
Just wondering, can we move this logic to `calypso/state/staging-site/selectors`?